packages/design/src/tokens/dark.tokens.json

```
@@ -19,248 +19,313 @@
-                "$value": "{color.base.yellow-.700}"
+                "$value": "{color.base.yellow-.700}",
+                "$description": "Use for the surface of an element that has warning status."
             },
             "onSurface": {
-                "$value": "{color.base.yellow-.200}"
+                "$value": "{color.base.yellow-.200}",
+                "$description": "Use for text or iconography that sits overtop a warning background."
             }
         },
         "success": {
-            "$value": "{color.base.green-.500}"
+            "$value": "{color.base.green-.500}",
+            "$description": "No action required; an action has completed successfully."
         },
         "success-": {
             "surface": {
-                "$value": "{color.base.green-.800}"
+                "$value": "{color.base.green-.800}",
+                "$description": "Use for the surface of an element that has success status."
             },
             "onSurface": {
-                "$value": "{color.base.green-.300}"
+                "$value": "{color.base.green-.300}",
+                "$description": "Use for text or iconography that sits overtop a success background."
             }
         },
         "informative": {
-            "$value": "{color.base.lightBlue-.600}"
+            "$value": "{color.base.lightBlue-.600}",
+            "$description": "No action required; but helpful to know about."
         },
         "informative-": {
             "surface": {
-                "$value": "{color.base.lightBlue-.800}"
+                "$value": "{color.base.lightBlue-.800}",
+                "$description": "Use for the surface of an element that has informative status."
             },
             "onSurface": {
-                "$value": "{color.base.lightBlue-.300}"
+                "$value": "{color.base.lightBlue-.300}",
+                "$description": "Use for text or iconography that sits overtop an informative background."
             }
         },
         "inactive": {
-            "$value": "{color.base.blue-.500}"
+            "$value": "{color.base.blue-.500}",
+            "$description": "No action required; not part of an active workflow."
         },
         "inactive-": {
             "surface": {
-                "$value": "{color.base.blue-.700}"
+                "$value": "{color.base.blue-.700}",
+                "$description": "Use for the surface of an element that has inactive status."
             },
             "onSurface": {
-                "$value": "{color.base.blue-.200}"
+                "$value": "{color.base.blue-.200}",
+                "$description": "Use for text or iconography that sits overtop an inactive background."
             }
         },
         "heading": {
-            "$value": "{color.base.blue-.100}"
+            "$value": "{color.base.blue-.100}",
+            "$description": "Headings have a bold, dark color to cement their hierarchy."
```

```suggestion
            "$description": "Headings have high contrast against the application surface to cement their hierarchy."
```

This is an oversight on my part on the content side, just going to make a few of these suggestions where we're referencing "dark" or "light" directly in the semantic tokens to be more _semantic_ and less _descriptive_. (I'll make these changes in the Figma as well!)